tests/test_recalibrate.yml

```diff
@@ -52,7 +58,13 @@
       should_exist: false
     - path: results/csv/recalibrated.csv
       should_exist: false
-    - path: results/reports/qualimap/test/test.recal
+    - path: results/reports/mosdepth/test/test.recal.mosdepth.global.dist.txt
+      should_exist: false
```

Comment:
Just a question, so is here testing that the files are not existing? Or what is the `should_exist` doing?